modules/howtos/pages/provisioning-cluster-resources.adoc

```diff
@@ -20,11 +35,11 @@ NOTE: When using a Couchbase version earlier than 6.5, you must create a valid B
 The https://docs.couchbase.com/sdk-api/couchbase-ruby-client/Couchbase/Management/BucketManager.html[BucketManager] interface may be used to create and delete buckets from the Couchbase cluster.
 It is instantiated through the `#create_bucket()` method.
 
-include::6.5@sdk:shared:partial$flush-info-pars.adoc[tag=update-bucket-warning]
+include::6.6@sdk:shared:partial$flush-info-pars.adoc[tag=update-bucket-warning]
 
 ////
 Here is the list of parameters available:
-
+// Ruby SDK doesn't have a good equivalent list...
 |====
```

Comment:
The settings are documented here:
https://github.com/couchbase/couchbase-ruby-client/blob/8a042f28e8cc00423edd880164e1db132c687a96/lib/couchbase/management/bucket_manager.rb#L203-L254

Reply:
Or even better: https://docs.couchbase.com/sdk-api/couchbase-ruby-client/Couchbase/Management/BucketSettings.html